packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -42,6 +42,7 @@ class HorizontalOverflow extends Foundation<
     public static defaultProps: Partial<HorizontalOverflowProps> = {
         managedClasses: {},
         nextItemPeek: 50,
+        fixedHeight: null,
```

Comment:
Does this really need to be set to null in the `defaultProps`?  Does omitting this and making the value undefined by default have the same result?

If for whatever reason we *do* need to initialize to null, I think the prop interface should reflect that it can be `null`

Reply:
Having this set to null means it will never be of value `undefined`, I'll update the prop definition to be number | null